### PR TITLE
 docs: describe some default values 

### PR DESCRIPTION
**Pull request by tokyo4j**

### CODE REVIEW

File: docs/labwc-theme.5.scd [hunk 1]
```diff
@@ -60,31 +60,33 @@ labwc-config(5).
 	Default is 0.
 
 *window.active.border.color*
-	Border color of active window.
+	Border color of active window. Default is #e1dedb.
```
Comment (tokyo4j) [inline docs/labwc-theme.5.scd:63]: https://github.com/labwc/labwc/blob/c2928027becba26f058b544d4bd8c72895c8d7cf/src/theme.c#L566

File: docs/labwc-theme.5.scd [hunk 2]
```diff
@@ -60,31 +60,33 @@ labwc-config(5).
...
 
 *window.inactive.border.color*
-	Border color of inactive window.
+	Border color of inactive window. Default is #f6f5f4.
 
 *window.active.indicator.toggled-keybind.color*
 	Status indicator for the ToggleKeybinds action. Can be set to the same
 	value as set for window.active.border.color to disable the status
-	indicator.
+	indicator. Default is #ff0000.
```
Comment (tokyo4j) [inline docs/labwc-theme.5.scd:71]: https://github.com/labwc/labwc/blob/c2928027becba26f058b544d4bd8c72895c8d7cf/src/theme.c#L567

File: docs/labwc-theme.5.scd [hunk 3]
```diff
@@ -60,31 +60,33 @@ labwc-config(5).
 	Default is 0.
 
 *window.active.border.color*
-	Border color of active window.
+	Border color of active window. Default is #e1dedb.
 
 *window.inactive.border.color*
-	Border color of inactive window.
+	Border color of inactive window. Default is #f6f5f4.
```
Comment (tokyo4j) [inline docs/labwc-theme.5.scd:66]: https://github.com/labwc/labwc/blob/c2928027becba26f058b544d4bd8c72895c8d7cf/src/theme.c#L567

File: docs/labwc-theme.5.scd (hunk 2 above)
Comment (tokyo4j) [inline docs/labwc-theme.5.scd:71]: https://github.com/labwc/labwc/blob/c2928027becba26f058b544d4bd8c72895c8d7cf/src/theme.c#L569

File: docs/labwc-theme.5.scd [hunk 4]
```diff
@@ -60,31 +60,33 @@ labwc-config(5).
...
 
 *window.active.indicator.toggled-keybind.color*
 	Status indicator for the ToggleKeybinds action. Can be set to the same
 	value as set for window.active.border.color to disable the status
-	indicator.
+	indicator. Default is #ff0000.
 
 *window.active.title.bg.color*
 	Background color for the focused window's titlebar.
+	Default is #e1dedb.
```
Comment (tokyo4j) [inline docs/labwc-theme.5.scd:75]: https://github.com/labwc/labwc/blob/c2928027becba26f058b544d4bd8c72895c8d7cf/src/theme.c#L571

File: docs/labwc-theme.5.scd [hunk 5]
```diff
@@ -60,31 +60,33 @@ labwc-config(5).
...
 
 *window.inactive.title.bg.color*
 	Background color for non-focused windows' titlebars.
+	Default is #f6f5f4.
```
Comment (tokyo4j) [inline docs/labwc-theme.5.scd:79]: https://github.com/labwc/labwc/blob/c2928027becba26f058b544d4bd8c72895c8d7cf/src/theme.c#L572

File: docs/labwc-theme.5.scd [hunk 6]
```diff
@@ -60,31 +60,33 @@ labwc-config(5).
...
 
 *window.active.label.text.color*
-	Text color for the focused window's titlebar.
+	Text color for the focused window's titlebar. Default is #000000.
```
Comment (tokyo4j) [inline docs/labwc-theme.5.scd:82]: https://github.com/labwc/labwc/blob/c2928027becba26f058b544d4bd8c72895c8d7cf/src/theme.c#L574

File: docs/labwc-theme.5.scd [hunk 7]
```diff
@@ -60,31 +60,33 @@ labwc-config(5).
...
 
 *window.inactive.label.text.color*
-	Text color non-focused windows' titlebars.
+	Text color non-focused windows' titlebars. Default is #000000.
```
Comment (tokyo4j) [inline docs/labwc-theme.5.scd:85]: https://github.com/labwc/labwc/blob/c2928027becba26f058b544d4bd8c72895c8d7cf/src/theme.c#L575

File: docs/labwc-theme.5.scd [hunk 8]
```diff
@@ -108,11 +110,11 @@ labwc-config(5).
 
 *window.active.button.unpressed.image.color*
 	Color of the images in titlebar buttons in their default, unpressed,
-	state. This element is for the focused window.
+	state. This element is for the focused window. Default is #000000.
 
 *window.inactive.button.unpressed.image.color*
 	Color of the images in titlebar buttons in their default, unpressed,
-	state. This element is for non-focused windows.
+	state. This element is for non-focused windows. Default is #000000.
```
Comment (tokyo4j) [inline docs/labwc-theme.5.scd:117]: https://github.com/labwc/labwc/blob/c2928027becba26f058b544d4bd8c72895c8d7cf/src/theme.c#L583-L589

File: docs/labwc-theme.5.scd [hunk 9]
```diff
@@ -173,39 +175,39 @@ all are supported.
 	Default is 4.
 
 *menu.items.bg.color*
-	Background color of inactive menu items.
+	Background color of inactive menu items. Default is #fcfbfa.
```
Comment (tokyo4j) [inline docs/labwc-theme.5.scd:178]: https://github.com/labwc/labwc/blob/c2928027becba26f058b544d4bd8c72895c8d7cf/src/theme.c#L603

File: docs/labwc-theme.5.scd [hunk 10]
```diff
@@ -173,39 +175,39 @@ all are supported.
...
 
 *menu.items.text.color*
-	Text color of inactive menu item.
+	Text color of inactive menu item. Default is #000000.
```
Comment (tokyo4j) [inline docs/labwc-theme.5.scd:181]: https://github.com/labwc/labwc/blob/c2928027becba26f058b544d4bd8c72895c8d7cf/src/theme.c#L604

File: docs/labwc-theme.5.scd [hunk 11]
```diff
@@ -173,39 +175,39 @@ all are supported.
...
 
 *menu.items.active.bg.color*
-	Background color of active menu items.
+	Background color of active menu items. Default is #e1dedb.
```
Comment (tokyo4j) [inline docs/labwc-theme.5.scd:184]: https://github.com/labwc/labwc/blob/c2928027becba26f058b544d4bd8c72895c8d7cf/src/theme.c#L605

File: docs/labwc-theme.5.scd [hunk 12]
```diff
@@ -173,39 +175,39 @@ all are supported.
...
 
 *menu.items.active.text.color*
-	Text color of active menu item.
+	Text color of active menu item. Default is #000000.
```
Comment (tokyo4j) [inline docs/labwc-theme.5.scd:187]: https://github.com/labwc/labwc/blob/c2928027becba26f058b544d4bd8c72895c8d7cf/src/theme.c#L606

**Comment by johanmalm** [comment]
Thanks!

**Pull request merged**